app/Jobs/SendNewRecordingNotifications.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Friend;
+use App\Jobs\Job;
+use App\Phone\Exceptions\BlacklistedPhoneNumberException;
+use App\Phone\TwilioClient;
+use App\PhoneNumber;
+use App\Recording;
+use App\User;
+use Illuminate\Log\Writer as Logger;
+
+class SendNewRecordingNotifications extends Job
```

Comment:
Hm. Smushing them into one makes two clean classes into one kinda muddy class. What's the benefit we get here?